### PR TITLE
Prevent having spaces in signal's method in Connect Dialog

### LINKED ISSUE #39647: Space in connect dialog copied into script file causes error
**Godot version:**
3.2.1 release

**OS/device including version:**
Windows 10

**Issue description:**
Adding a space when connecting a method via GUI to a script throws a GDScript error after the name has been pasted into the script file.
bug-img

The name should either be trimmed or a warning should pop up.

**Steps to reproduce:**
- Select a node and double click on a signal to open the connect dialog
- Add a space somewhere in the method name and press 'Connect'
- The method name will be pasted into the GDScript file. When opening the file an error will appear (picture above)

**Minimal reproduction project:**
Can be done in a new project

### PR DESCRIPTION
**Pull request by dreamsComeTrue**
Fixes: #39647

### CODE REVIEW

File: editor/connections_dialog.cpp [hunk 1]
```diff
@@ -108,17 +108,26 @@ class ConnectDialogBinds : public Object {
...
-	if (dst_method->get_text() == "") {
+	String method_name = dst_method->get_text();
+
+	if (method_name == "") {
 		error->set_text(TTR("Method in target node must be specified."));
 		error->popup_centered();
 		return;
 	}
+
+	if (method_name.strip_edges().find_char(' ') > 0) {
```
Comment (akien-mga) [inline editor/connections_dialog.cpp:119]: There's more than just spaces making a string an invalid identifier. Use `String::is_valid_identifier` :)

Comment (dreamsComeTrue) [reply editor/connections_dialog.cpp]: You're correct! ;)

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**